x-pack/plugins/fleet/server/services/epm/packages/package_verification.ts

```diff
@@ -128,6 +130,13 @@ async function _verifyPackageSignature({
     verificationKeys: verificationKey,
     signature,
     message,
+    config: {
+      // See https://github.com/openpgpjs/openpgpjs/blob/d6145ac73eebcf66bdeb0873aa60fc49361e1aeb/src/message.js#L800-L809
```

Comment:
@elastic/kibana-security can you take a look?

Reply:
Thanks for the tag @jbudz! Using this config option is alright with us given the context. LGTM!